docs/abi-spec.rst

```diff
@@ -44,7 +44,7 @@ The following elementary types exist:
 
 - ``int<M>``: two's complement signed integer type of ``M`` bits, ``0 < M <= 256``, ``M % 8 == 0``.
 
-- ``address``: equivalent to ``uint160``, except for the assumed interpretation and language typing. For computing the function selector, ``address`` is used.
+- ``address``, ``address payable``: equivalent to ``uint160``, except for the assumed interpretation and language typing. For computing the function selector, ``address`` is used.
```

Comment:
I think we should not mention it here, because those are only the proper ABI types, i.e. it does not include `enum` or `contracts`. If we include it here, we also have to change the function signature / selector.

We could mention how Solidity types are mapped to ABI types at the bottom.

Reply:
I created #4950 to track this - I think this is better done in a separate PR.